packages/editor-ui/src/utils/nodeViewUtils.ts

```diff
@@ -303,10 +317,16 @@ export const getOutputNameOverlay = (
 			label.classList.add('node-output-endpoint-label');
 
 			label.setAttribute('data-endpoint-node-type', ep?.__meta?.nodeType);
```

Comment:
This attribute is no longer used -> can be removed

Reply:
Thanks, totally overlooked this one. Got removed.